### ISSUE TITLE
BUG - sev-3 - Fix areas with too much spacing in the app

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#8891]
## What happened?

QA noticed that there are areas in the app that are receiving too much spacing after paragraphs. Ticket #8799 was opened and some exploratory work was done. It was discovered that not only do we have too much spacing, but that our spacing is very inconsistent.

### Brea's findings:

> After doing some research, we are definitely inconsistent in our paragraph spacing. 
> 
> Initially, I took some time to review the WCAG guidance and additional guidance from other sources for accessibility best practices in native mobile apps (including MCAG). All sources to recommend line-height at 1.5x the font size and 2x the font size for paragraph spacing, which is what was recommended during our WCAG updates. However, this does not seem to be what we are doing throughout the app (likely due to new components being implemented) and we are seeing areas (like in the example above), with inconsistent spacing and too much spacing between elements.
> 
> I did some [exploration in Figma](https://www.figma.com/design/BVfWDc8vinwx8kRc1LoqGl/%F0%9F%8F%96-Brea's-Sandbox?node-id=1296-883&t=9HWuO4tZoiKpTuFQ-0) and discovered that many of the elements on the screens shown in the example have as much as 67px between elements, with the average amount of spacing being 64px. This should be 40px and should not increase even if links are present.
> 
> I [included a side-by-side comparison](https://www.figma.com/design/BVfWDc8vinwx8kRc1LoqGl/%F0%9F%8F%96-Brea's-Sandbox?node-id=1297-4748&t=9HWuO4tZoiKpTuFQ-4) in the Figma file which shows what is in the app today vs. what we should actually have in the app. I also took a look at some of the other areas of the app (mostly focusing in on the health area where we have longer blocks of copy) and included [some examples of what the spacing should be](https://www.figma.com/design/BVfWDc8vinwx8kRc1LoqGl/%F0%9F%8F%96-Brea's-Sandbox?node-id=1297-5334&t=9HWuO4tZoiKpTuFQ-4). 
> 
> I've talked to @wavelaurenrussell quite a bit about the spacing inconsistencies and I know that this is something that she was looking to resolve in her recent work on appointments, but this is really an app-wide issue that we need to resolve. 
> 
> The purpose of the additional spacing after paragraphs is a way to break up large, flowing blocks of copy from other copy to improve legibility, but also as a way to help visually group content. While we don't always have to have exactly 40px between elements / below paragraphs, but it should look consistent visually, which is not what we are seeing in the app today.

## Desired behavior

Desired spacing is 40px (or the equivalent in React Native) after longer blocks of copy (2-3+ lines at the default zoom and text sizing on a mobile device). It doesn't not always have to be exactly 40px, but the spacing should visually look consistent.

## Acceptance Criteria

- [ ] Compare the current spacing in the app to what it should be using the [examples in Figma](https://www.figma.com/design/BVfWDc8vinwx8kRc1LoqGl/%F0%9F%8F%96-Brea's-Sandbox?node-id=1297-4749&t=Rhx4Z7K9Cf6OUNiE-4).
- [ ] Determine what is causing the additional spacing.
- [ ] Resolve the spacing issues on the following screens:
     - [x] Login information screen
     - [x] Veterans Crisis Line panel
     - [x] Profile > How to fix an error in your legal name
     - [x] Profile > How to fix an error in your date of birth
     - [x] Profile > What to know before you decide to share your gender identity
     - [x] Profile > How we use your contact information
     - [x] Profile > What if my military service information doesn't look right?
- [ ] Send to @brea11y to review.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

_One of the examples of what is currently in the app vs. the spacing we should have in the app:_
<img width="591" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMVNKQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--af2de97c4ce54f5bcde08997ee56dd197abc9462/Screenshot%202024-06-24%20at%2012.03.37%E2%80%AFPM.png" alt="side-by-side comparison of the paragraph spacing in the app today versus what we should have" />

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by dumathane** [comment]
Discussed on 7/3: Brea will come to next pending handoff with a full global list to investigate and fix. Maybe want to create a H+B 'checkall' ticket as well.

**Comment by brea11y** [comment]
The Figma file has been updated with the Global screens that need review / possible updating. I have included examples in the file of what they should look like with proper spacing and updated the AC to include the full list of screens.

**Comment by theodur** [comment]
For QA purposes, I wanted to point out that the padding under the bullet point section on the `LoaGate` screen (Sign in) and the `What to know before you decide to share your gender identity screen` (Profile help) screen will be fixed by the changes in https://github.com/department-of-veterans-affairs/va-mobile-app/pull/9115, So the padding in those 2 spots will still not look correct in my PR. If it makes it easier to review, we can always want to wait for that one to be merged (it's currently in QA) before QA'ing this one

**Comment by TKDickson** [comment]
I haven't checked login info yet (awaiting #9115), but all of the rest of them looked at least 1) better and 2) afaik, matching the examples in the linked Figma. I'm still seeing the icon wrapping issue in #9054 which I expected, since that hasn't been worked yet.

Except this still looks like more spacing than intended (and, since it's nowhere near needing to wrap, I don't think it's the same thing as #9054). @theodur can you take a look?
<img width="715" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1lTQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6049ed94a04ffc235a64b3f2dc8d42ab2d86ff4b/image.png" alt="image.png" />

**Comment by TKDickson** [comment]
Marking as blocked for now, will loop back once 9115 is in develop.

**Comment by TKDickson** [comment]
This is likely to be blocked for a while yet - claims 8 steps is the top priority in H&B, and with both Eng out they're moving more slowly than usual. QA is aware of the blocker.

**Comment by TKDickson** [comment]
Blocking ticket has been closed; removing label (and hopefully wrapping up today)

**Comment by TKDickson** [comment]
Confirmed improved spacing on login and gender identity help (two remaining screens). I will be approving this PR, and also letting Brea know that the majority of spacing fixes are through so it's ready for review.

**Comment by TKDickson** [comment]
Doesn't require further testing in develop; closing
